<launch>
  <arg name="rviz" default="false" />
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
      args="-d $(find laser_slam)/rviz/tracking1.rviz" />
  </group>
  <node pkg="laser_slam" type="tracking1" name="tracking1" output="screen"></node>
</launch>
